.github/workflows/change-tag-name.yml

```
@@ -1,7 +1,6 @@
 on:

```

Consider adding a name
```suggestion
name: Add tag without v

on:

```